vars/daosPackagesVersion.groovy

```diff
@@ -0,0 +1,96 @@
+// vars/daosPackagesVersion.groovy
+
+/**
+ * daosPackagesVersion.groovy
+ *
+ * daosPackagesVersion variable
+ */
+
+/**
+ * Method to return the version of the DAOS packages
+ */
+
+import groovy.transform.Field
+
+@Field static rpm_version_cache = ""
+
+String daos_latest_version(String next_version) {
+    String v = sh(label: "Get RPM packages version",
+                  script: 'repoquery --repofrompath=daos,' + env.REPOSITORY_URL +
+                           env.DAOS_STACK_EL_7_LOCAL_REPO + '''/ \
```

Comment:
See https://github.com/daos-stack/pipeline-lib/pull/207

For CENTOS, the new repos are:
DAOS-STACK-EL-7-STABLE-REPO  - Contains signed releases, will contain tagged builds in the future, and the DAOS_STACK_EL_7_LOCAL-REPO contents.

or

DAOS_STACK_EL_7_RELEASE_REPO - This only contains signed released RPMS

or

DAOS_STACK_EL_7_DEV-REPO - This is what ever we put in to if for special PR testing.

Reply:
Is this still true after our discussion about having group repos with signed and unsigned packages in it?

Reply:
Yes for this case.

For yum repos, we only want to be reading from group repos in the artifacts server, not the local repos.

The DAOS-STACK-EL-7-STABLE-REPO and DAOS-STACK-EL-7-DEV-REPO will need all gpg checks disabled as they are all locally built content.

The DAOS-STACK-EL-7-RELEASE-REPO  can have package gpg-checks enabled, but not repo gpg-checks enabled since our artifact server does not support repo gpg-checks for yum repos.

Reply:
So then, given that I have no clue what repos are what any more, what should I be using here?